<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R. Reucher</author>
 <class>MiniWebBrowser</class>
 <widget class="QWidget" name="MiniWebBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini Web Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="3" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frameBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frameSearch">
     <layout class="QHBoxLayout" name="horizontalLayoutFrameSearch">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter search string</string>
        </property>
        <property name="placeholderText">
         <string>Enter search string</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonPrevious">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Find previous occurrence</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/left_action.png</normaloff>:/data/img/left_action.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonNext">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Find next occurrence</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/right_action.png</normaloff>:/data/img/right_action.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonCaseSensitive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Case-sensitive</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/characters.png</normaloff>:/data/img/characters.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonHighlight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Highlight matches</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/highlight.png</normaloff>:/data/img/highlight.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frameUrl">
     <layout class="QHBoxLayout" name="horizontalLayoutFrameUrl">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButtonBack">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Go back</string>
        </property>
        <property name="statusTip">
         <string>Go back</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/back.png</normaloff>:/data/img/back.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonForward">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Go forward</string>
        </property>
        <property name="statusTip">
         <string>Go forward</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/forward.png</normaloff>:/data/img/forward.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonReload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Reload current page</string>
        </property>
        <property name="statusTip">
         <string>Reload current page</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/reload.png</normaloff>:/data/img/reload.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Stop loading of current page</string>
        </property>
        <property name="statusTip">
         <string>Stop loading of current page</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/stop_browser.png</normaloff>:/data/img/stop_browser.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonHome">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Go home (first page)</string>
        </property>
        <property name="statusTip">
         <string>Go home (first page)</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/home.png</normaloff>:/data/img/home.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxZoom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Set zoom factor</string>
        </property>
        <property name="statusTip">
         <string>Set zoom factor</string>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>800</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonToggleSearchBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Toggle display of the search bar</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/find.png</normaloff>:/data/img/find.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxURL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter current URL</string>
        </property>
        <property name="statusTip">
         <string>Enter current URL</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonLoad">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Load URL</string>
        </property>
        <property name="statusTip">
         <string>Load URL</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/yes.png</normaloff>:/data/img/yes.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>toolButtonBack</tabstop>
  <tabstop>toolButtonForward</tabstop>
  <tabstop>toolButtonReload</tabstop>
  <tabstop>toolButtonStop</tabstop>
  <tabstop>toolButtonHome</tabstop>
  <tabstop>spinBoxZoom</tabstop>
  <tabstop>toolButtonToggleSearchBar</tabstop>
  <tabstop>comboBoxURL</tabstop>
  <tabstop>toolButtonLoad</tabstop>
  <tabstop>lineEditSearch</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
